narayana/tools/src/main/java/io/narayana/perf/jmh/Benchmark.java

```diff
@@ -74,16 +74,21 @@ public void setPrevious(Benchmark previous) {
 
 	@Override
 	public String toString() {
-		if (previous != null) {
-			double percentChange = regression == 0 ? 0.0 : (score / previous.score - 1) * 100.0;
-			String changeType = regression == -1 ? "regression" : regression == 0 ? "no change" : "improvement";
-			return String.format("%s: %f vrs %f (%f%%: %s)%n", benchmark, score, previous.score, percentChange,
-					changeType);
-
-		} else {
-			return String.format("%s: %f%n", benchmark, score);
-		}
-	}
+        if (previous != null) {
+            double percentChange = regression == 0 ? 0.0 : (score / previous.score - 1) * 100.0;
+            String changeType = regression == -1 ? "regression" : regression == 0 ? "no change" : "improvement";
+            if (regression == 0) {
```

Comment:
I was just referring to the formatting of the new code and this reformats more than the code added but I don't have an objection to method being reformatted as it's in the performance repo (and not the main narayana repo say) so it's OK from my point of view. Thanks, Tom